src/hazelcore/Action_Exp.re

```diff
@@ -1912,9 +1979,8 @@ and syn_perform_operand =
         };
       }
     }
-  | (Init, _) => failwith("Init action should not be performed.")
-  };
-}
+  | (Construct(SCloseBraces), _) => Failed
```

Comment:
why does SCloseBraces but not SCloseParens have a fallthrough case like this?

Reply:
The one for SCloseParens was explicitly put as (Construct(SCloseParens), CursorE(_)) => Failed, doing the same for this one too